### PR TITLE
Desktop version portfolio complete

### PR DESCRIPTION
**Pull request by sulaimanahmadi**
**Add following things.**

- Use of flexbox and grid.
- Use media queries.
- Use Html and CSS.

### CODE REVIEW

File: desktop.css [hunk 1]
```diff
@@ -159,12 +159,31 @@ body {
   flex-grow: 0;
 }
 
+.content-data > h2 {
```
Comment (joskalenda) [inline desktop.css:164]: - [ ] Your application is not responsive, @sulaimanahmadi, I think you should better use the `min-width` property and make sure you style your app accordingly as for now, it is not respecting any of your CSS in big screens

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @sulaimanahmadi , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sulaimanahmadi** [comment]
Hello Dear ***@***.****
*I have changed some code and attached the video so kindly see to confirm
myself about it.*

Best regards

File: style.css [hunk 2]
```diff
@@ -8,6 +8,7 @@ body {
   background-color: white;
   margin: 0;
   padding: 0;
+  width: 100%;
 }
 
 .container {
```
Comment (tresorsawasawa) [inline style.css]: - [ ] As mentioned by the previous reviewer and for the project requirements, kindly make sure your application is responsive and the design sticks as much as possible to the one from your chosen design. Below you can see hoew your application behaviors when changing the screen size.
git

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @sulaimanahmadi 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sulaimanahmadi** [comment]
Hello Dear  ***@***.****

*On behalf of my pull request, I requested the previous code reviewer also
to check my attached video therefore I made the desktop version portfolio
on a desktop screen and it is responsive so kindly watch my video.*

*best regards,*

**Review by Cabraham1 — APPROVED** [review]
Hi @sulaimanahmadi  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**